oak_functions_enclave_service/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "oak_functions_enclave_service"
```

Comment:
considered just moving this logic into `enclave_apps/oak_functions_enclave_app/src/lib.rs`, akin to `oak_functions_containers_app/src/lib.rs`. But it leads to some issues surrounding prost versions, since we use a different lockfile in the `enclave_apps` dir.